### ISSUE TITLE
`balena devices supported` is undocumented

### ISSUE THREAD

**Issue by xginn8** [balena-io/balena-cli#1794]
- **balena CLI version:** 
```
❯ balena version
11.32.1
```
There is no mention in the `balena devices --help`, but the command exists (cc @cmfcruz):

```
❯ balena devices --help          
Usage: devices

Use this command to list all devices that belong to you.

You can filter the devices by application by using the `--application` option.

Examples:

        $ balena devices
        $ balena devices --application MyApp
        $ balena devices --app MyApp
        $ balena devices -a MyApp

Options:

    --application, -a, --app <application> application name                   
```

```
❯ balena devices supported --help
List the supported device types (like 'raspberrypi3' or 'intel-nuc').

USAGE
  $ balena devices supported

OPTIONS
  -h, --help      show CLI help
  -j, --json      produce JSON output instead of tabular output
  -v, --verbose   add extra columns in the tabular output (ALIASES, ARCH, STATE)
  --discontinued  include "discontinued" device types

DESCRIPTION
  List the supported device types (like 'raspberrypi3' or 'intel-nuc').

  The --verbose option adds extra columns/fields to the output, including the
  "STATE" column whose values are one of 'beta', 'released' or 'discontinued'.
  However, 'discontinued' device types are only listed if the '--discontinued'
  option is used.

  The --json option is recommended when scripting the output of this command,
  because the JSON format is less likely to change and it better represents data
  types like lists and empty strings (for example, the ALIASES column contains a
  list of zero or more values). The 'jq' utility may be helpful in shell scripts
  (https://stedolan.github.io/jq/manual/).

EXAMPLES
  $ balena devices supported
  $ balena devices supported --verbose
  $ balena devices supported -vj
```

**Comment by pdcastro** [comment]
Connects to:
* #694 Getting help on nested commands 
* #1358 'balena help' hides too much behind the verbose flag 
* #1343 "balena help XXX": command not found

**Comment by pdcastro** [comment]
FYI, the full command list is available with `balena help -v`:
```
$ balena help -v
...
    devices supported                     list the supported device types (like 'raspberrypi3' or 'intel-nuc')
```

**Comment by srlowe** [comment]
@pdcastro Is there any point in keeping this issue open if we already have open ones that describe the same problems?

**Comment by pdcastro** [comment]
OK, I think this one can be closed because:
* All subcommands are undocumented (not just `devices supported`) in the sense reported in this issue.
* I've listed three closely related issues above, and I have bumped their priority a bit in the [ordered backlog](https://github.com/balena-io/balena-cli/milestone/2).
* Even when closed, this issue is still searchable and users can still add comments to it.
